po/uk.po

```diff
@@ -712,7 +712,6 @@ msgid "Browser returned error code %i"
 msgstr "Браузер повернув код помилки %i"
 
 #: src/feedlistformaction.cpp:299 src/itemlistformaction.cpp:487
-#, fuzzy
 msgid "Do you really want to mark this feed as read (y:Yes n:No)? "
 msgstr "Ви впевнені, що хочете відмітити усі теми як прочитані (y:Так n:Ні)? "
```

Comment:
I still don't know any Ukrainian, but is this translation correct? The Russian translation got updated and in the German version there was a slight difference (all feeds vs. this feed) as well.

Reply:
Oh right, the translation says "all feeds" rather than "this feed". Thanks, will fix in a second!

Reply:
Probably good now. ;-)